Dockerfile

```diff
@@ -1,84 +1,48 @@
-FROM centos/httpd:latest
-MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
+FROM registry.access.redhat.com/ubi8/ubi-init
+MAINTAINER ManageIQ https://github.com/ManageIQ/container-httpd
 
-## Set build ARGs
 ARG DBUS_API_REF=master
 
-## Systemd
-ENV container oci
-
-## Atomic/OpenShift Labels
 LABEL name="auth-httpd" \
       vendor="ManageIQ" \
       url="http://manageiq.org/" \
       summary="httpd image with external authentication" \
-      description="An httpd image which includes packages and configuration necessary for handling external authentication." \
-      io.k8s.display-name="Httpd with Authentication" \
-      io.k8s.description="An httpd image which includes packages and configuration necessary for handling external authentication." \
-      io.openshift.expose-services="80:http" \
-      io.openshift.tags="httpd"
-
-## To cleanly shutdown systemd, use SIGRTMIN+3
-STOPSIGNAL SIGRTMIN+3
-
-## Install EPEL repo, yum necessary packages for the build without docs, clean all caches
-RUN yum -y install centos-release-scl-rh && \
-    yum -y install --setopt=tsflags=nodocs mod_ssl                      \
-                                           && \
+      description="An httpd image which includes packages and configuration necessary for handling external authentication."
+
+RUN dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-repos-8.1-1.1911.0.8.el8.x86_64.rpm http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm && \
+    dnf -y --disableplugin=subscription-manager module enable mod_auth_openidc && \
```

Comment:
just curious, why does mod_auth_openidc need a module enable, but the other mod_auth ones do not ?

Reply:
No idea. This is just what I needed to do to get access to the packages. I think this is also what @simaishi had to do for RHEL/CentOS 8.

Reply:
@abellotti mod_auth_openidc module doesn't have default stream defined. When there is no default, you have to enable it before using it. Also, other mod_auth_* are not yum modules but standalone RPMs, so no need to enable.

Reply:
Ahh ok, makes sense.  Thanks!!